site/pages/search-results-contextual-en.hbs

```diff
@@ -0,0 +1,111 @@
+---
+{
+	"title": "Search results",
+	"language": "en",
+	"altLangPrefix": "search",
+	"pageType": "search",
+	"pageclass": "page-type-search",
+	"breadcrumb": [
+		{ "title": "Home", "link": "https://www.canada.ca/en.html" }
+	],
+	"dateModified": "2018-12-21",
+	"share": "false",
+	"deptfeature": false
+}
+---
+<!--Note: visual example with styles applied: https://gc-proto.github.io/student-loans/scenarioA/services/search-results-contextual-visual.html-->
+<!--Note 2: this template should only be used for contextual search results from a single institution-->
+<!--Note 3: until radio buttons pattern works, please simply include 'All government of Canada websites' link in the well, under the search box-->
+<div class="row profile">
```

Comment:
What is the use of the CSS ```profile```?

That ```div.row``` followed by ```div.col-md-12``` are probably useless in that design

Reply:
No idea will remove row 19 and the closing div

Reply:
ok

Reply:
don't forget to remove the div with the column that is after ```<div class="col-md-12">```

Reply:
Can't see anything with a column inside the <div class="col-md-12">